### PR TITLE
Sign the VSIX

### PR DESCRIPTION
**Pull request by jgsogo**
Supersedes https://github.com/conan-io/conan-vs-extension/pull/102, we need to run this branch in the same repo so the secret env variables from Appveyor are resolved.

The CI is working, and the extension is signed, but it fails to install. Almost sure that the error is because the certificate must be available on the internet.

### CODE REVIEW

**Comment by SSE4** [comment]
need to merge conflicts

File: Conan.VisualStudio/packages.config [hunk 1]
```diff
@@ -27,6 +27,7 @@
   <package id="Microsoft.VisualStudio.Utilities" version="15.0.26201" targetFramework="net461" />
   <package id="Microsoft.VisualStudio.Validation" version="15.3.32" targetFramework="net461" />
   <package id="Microsoft.VSSDK.BuildTools" version="15.1.192" targetFramework="net461" developmentDependency="true" />
+  <package id="Microsoft.VSSDK.VsixSignTool" version="16.1.28916.169" targetFramework="net461" />
```
Comment (SSE4) [inline Conan.VisualStudio/packages.config]: probably need to downgrade this one

**Comment by SSE4** [comment]
is it okay to have cert file in public (in source code)? does it mean anyone will be able to sign his our extension as conan's?

**Comment by jgsogo** [comment]
You need a password to use that file (check how I'm using it in the CI), but this is my first attempt to sign anything so maybe I'm wrong

**Comment by SSE4** [comment]
@jgsogo again conflicts

**Comment by AbrilRBS** [comment]
Closing as we've just released a new vs2022 compatible version that differs from this base, which will be the version we'll be supporting in the future, feel free to check it out in https://marketplace.visualstudio.com/items?itemName=conan-io.conan-vs-extension&ssr=false#overview

**Pull request closed without merging**